proto/sf/ethereum/transform/v1/transforms.proto

```diff
@@ -3,6 +3,12 @@ syntax = "proto3";
 package sf.ethereum.transform.v1;
 option go_package = "github.com/streamingfast/sf-ethereum/types/pb/sf/ethereum/transform/v1;pbtransform";
 
+// Log and CallTo Filters, applied as 'inclusive OR'
+message CombinedFilter {
```

Comment:
Personally, I don't find the name super appealing. But I can understand it would become a bit long like `CallAndLogCombinedFilter`, at least the comment will help.